compiler/semexprs.nim

```diff
@@ -2426,8 +2525,7 @@ proc semExpr(c: PContext, n: PNode, flags: TExprFlags = {}): PNode =
     # handling of sym choices is context dependent
     # the node is left intact for now
     discard
-  of nkStaticExpr:
-    result = semStaticExpr(c, n)
+  of nkStaticExpr: result = semStaticExpr(c, n[0])
```

Comment:
This is a style violation, please pass ``n`` to ``semStaticExpr``

Reply:
Not possible.

Notice how `semStaticExpr` is sometimes called with `n[0]` and sometimes with `n[1]`. I guess the name could have been different to avoid this "stylistic violation", but `semStaticExpr` differs from `evalStaticExpr` in the sense that it first performs a `sem` pass with `semExpr`, so I'm not sure what other name could have been more appropriate.

Reply:
Ok.